### PR TITLE
New version: Corel.WinZip version 28.0.15620

### PR DESCRIPTION
**Pull request by PckgrBot**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/120727)

### CODE REVIEW

**Comment by Trenly** [comment]
```
Action start 17:43:41: WebView2NotInstalledBis.
MSI (s) (BC:14) [17:43:41:398]: Product: WinZip 28.0 -- WebView2 is not installed. Please install the latest version of the ”WebView2 x64 Runtime” from www.catalog.update.microsoft.com

WebView2 is not installed. Please install the latest version of the ”WebView2 x64 Runtime” from www.catalog.update.microsoft.com
Action ended 17:43:41: WebView2NotInstalledBis. Return value 3.
Action ended 17:43:41: INSTALL. Return value 3.
```

File: manifests/c/Corel/WinZip/28.0.15620/Corel.WinZip.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+PackageIdentifier: Corel.WinZip
+PackageVersion: 28.0.15620
+InstallerLocale: en-US
+InstallerType: msi
+ProductCode: '{CD95F661-A5C4-44F5-A6AA-ECDD91C2414B}'
+ReleaseDate: 2023-09-08
+AppsAndFeaturesEntries:
+- DisplayName: WinZip 28.0
+  UpgradeCode: '{CD95F661-A5C4-44F5-A6AA-ECDD91C240A3}'
```
Comment (Trenly) [inline manifests/c/Corel/WinZip/28.0.15620/Corel.WinZip.installer.yaml:12]:
```suggestion
  UpgradeCode: '{CD95F661-A5C4-44F5-A6AA-ECDD91C240A3}'
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.EdgeWebView2Runtime
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978335226 
2023-09-25 21:55:28.215 [CLI ] ShellExecute installer failed: 1603 Property(S): ErrorDialog = SetupError Property(S): S_ERROR_TITLE = WinZip Install Error Property(S): SecureCustomProperties = <long string> Property(S): WinZipErrorPageUrl = http://www.winzip.com/wzgate.cgi?lang=EN&url=www.winzip.com/net4update.html 

> MSI (s) (44:64) [21:55:27:885]: Product: WinZip 28.0 -- Installation operation failed. 
MSI (s) (44:64) [21:55:27:885]: Windows Installer installed the product. Product Name: WinZip 28.0. Product Version: 28.0.15620. Product Language: 1033. Manufacturer: Corel Corporation. Installation success or error status: 1603. 

Hi @PckgrBot,

The status `1603` often indicates a missing dependency.

**Comment by Trenly** [comment]
This is failing because the version of winget in the pipelines seems to be 1.4 and not 1.6

> 2023-09-25 21:55:13.857 [CORE] WinGet, version [1.4.10052], activity [{BD2A5759-AEE6-405E-8B9A-70642BCFC50C}]

**Pull request closed without merging**